flink-end-to-end-tests/test-scripts/test_glue_schema_registry.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```

Comment:
Have discussed this with @LinyuYao1021 and agreed to accept the `sh` test and fast follow migrate to the Java solution. We will raise a follow up Jira and I will perform the migration. Since the migration is mostly going to consist of adding support for Kinesalite, I will then look at the Kinesis e2e tests since they should be similar.

Reply:
https://issues.apache.org/jira/browse/FLINK-21391

Reply:
@LinyuYao1021 Please update [run-nightly-tests.sh](https://github.com/apache/flink/blob/master/flink-end-to-end-tests/run-nightly-tests.sh) to include this test